tests/Dockerfile.centos-6

```diff
@@ -2,9 +2,8 @@ FROM centos:6
 
 # Install Ansible
 RUN yum -y update; yum clean all;
-RUN yum -y install epel-release
-RUN yum -y install git ansible sudo
-RUN yum clean all
+RUN yum -y install epel-release; yum clean all;
+RUN yum -y install git ansible sudo; yum clean all;
```

Comment:
That's is not just a formatting change. That's a layer rule following.
A Dockerfile is not a script but, let me say a "layer-builder". Each RUN build a layer, and if you have a fat layer at one time, your image will be fat.
So the way to mind is to put into the same RUN the things to do and the clean-up. for example, add-repository, update, install and clean-up should be into one unique layer.

As an example, an excerpt from the official doc':
Because image size matters, (...) you should avoid doing things like:
```
ADD http://example.com/big.tar.xz /usr/src/things/
RUN tar -xJf /usr/src/things/big.tar.xz -C /usr/src/things
RUN make -C /usr/src/things all
```
And instead, do something like:
```
RUN mkdir -p /usr/src/things \
    && curl -SL http://example.com/big.tar.xz \
    | tar -xJC /usr/src/things \
    && make -C /usr/src/things all
```

source : https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy